lib/materialist/materializer/internals/dsl.rb

```diff
@@ -6,7 +6,7 @@ def materialize_link(key, topic: key)
           __materialist_options[:links_to_materialize][key] = { topic: topic }
         end
 
-        def capture(key, as: key)
+        def capture(key, as: [key].flatten.last)
```

Comment:
Just an styling opinion: I think if we change this to `capture(*key, as: key.last)` the changes are kept minimal on the other file too?